test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -229,9 +229,9 @@ bool UcxContext::listen(const struct sockaddr* saddr, size_t addrlen)
     return true;
 }
 
-void UcxContext::progress()
+void UcxContext::progress(unsigned count)
 {
-    progress_worker_event();
+    progress_worker_event(count);
```

Comment:
need to progress all functions in here "count" times, not only progress_worker_event

Reply:
@yosefe if so, then maybe revise https://github.com/openucx/ucx/pull/7178#discussion_r686142442 again? it contradicts with https://github.com/openucx/ucx/pull/7178#discussion_r686886869

Reply:
yes, seems like progressing all functions would not have different effect than when we have with BUSY_PROGRESS_COUNT
@huaxiangfan can you check if doing so (progressing all functions in progress() count times) gives any perf improment?
if not, we can try to progress only progress_worker_event() and progress_io_message() "count" times

Reply:
if ((count++ < BUSY_PROGRESS_COUNT) || timer_finished) {
                progress();    => progress(1)
i think we can make above code change without leading to any behavior change.

Reply:
Sure. let me do it and verify.

Reply:
A new commit for progressing both worker and io_message instead of all functions which could achieve better perf improvement and with lesser progress count.